portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,13 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload = "rotateImages()">
     <table>
         <tr>
-            <th>About Me</th>
-            <th>Past Projects</th>
-            <th>Travel!</th>
-            <th>Fortune Teller</th>
+            <th><a href=#"about-me">About Me</a></th>
```

Comment:
This one is different from the other 3, as in, # sign is outside the quotes, vs inside for the other items in the table. 

Even if they mean the same thing, I recommend using consistent format for all entries.